demo/rube_testbed/loadrube.js

```diff
@@ -37,7 +37,11 @@ function loadBodyFromRUBE(bodyJso, world) {
     else
         bd.set_position(new Box2D.b2Vec2(0,0));
 
-    bd.set_gravityScale(bodyJso.gravityScale || 1);
+    if (bodyJso['gravityScale'] != undefined) {
```

Comment:
If You mean bodyJso.gravityScale != undefined and bodyJso['gravityScale'] != undefined. Yes it no difference. I can update if You want.